readthedocs/urls.py

```diff
@@ -90,7 +90,8 @@
 api_urls = [
     re_path(r'^api/v2/', include('readthedocs.api.v2.urls')),
     # Keep `search_api` at root level, so the test does not fail for other API
-    re_path(r"^api/v2/search/$", include("readthedocs.search.api.v2.urls")),
+    path("api/v2/search/", include("readthedocs.search.api.v2.urls")),
+    path("api/v3/search/", include("readthedocs.search.api.v3.urls")),
```

Comment:
We should probably keep using re_path for consistency?

Reply:
We have started using `path` on new code https://github.com/readthedocs/readthedocs.org/pull/9628/files.

And actually, the v2 URL was changed to `path` in another PR (that's the conflict the GitHub is showing on this PR)

Reply:
It's just weird to have both of them in a single file. We should keep them all the same unless we do a bit migration of all  of them, otherwise it seems like there is a reason it's this way to readers.